### PR TITLE
New package: zerotier-one-1.1.6

### PR DESCRIPTION
**Pull request by the-maldridge**
zerotier is a novel p2p networking solution.

### CODE REVIEW

**Comment by Vaelatern** [comment]
You chose the name `zerotier` instead of `zerotierone`, what was your reasoning?

**Comment by the-maldridge** [comment]
From the discussion in IRC, it was zerotier when I first started using it.  Now it seems they call the upstream package zerotier-one.

**Comment by Vaelatern** [comment]
Separately from packaging interests, will you be trying to upstream the "patches" necessary here?

**Comment by the-maldridge** [comment]
see zerotier/ZeroTierOne#348

**Comment by Vaelatern** [comment]
Perhaps a patch to allow installation to `/usr/bin` instead of insisting on `/usr/sbin` would be accepted upstream?

File: srcpkgs/zerotier-one/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	sed -i 's:#include <sys/sysctl.h>::' ./attic/OSXEthernetTap.cpp.pcap-with-bridge-test
+	sed -i 's:#include <sys/sysctl.h>::' ./attic/OSXEthernetTap.cpp.utun-work-in-progress
+	sed -i 's:#include <sys/sysctl.h>::' ./ext/libnatpmp/getgateway.c
+	sed -i 's:#include <sys/sysctl.h>::' ./osdep/BSDEthernetTap.cpp
+	sed -i 's:#include <sys/sysctl.h>::' ./osdep/ManagedRoute.cpp
+	sed -i 's:#include <sys/sysctl.h>::' ./osdep/OSXEthernetTap.cpp
+}
+
+do_build() {
+	make all ZT_\ENABLE_\NETWORK_\CONTROLLER=1
```
Comment (Vaelatern) [inline srcpkgs/zerotier-one/template]: Why all the backslashes?

Comment (the-maldridge) [inline srcpkgs/zerotier-one/template]: They are specified as part of the token in the documentation.  Its probably safe to remove them, but the token is specified in the readme with the slashes in place.

Comment (Vaelatern) [inline srcpkgs/zerotier-one/template]: Unable to see the token at all on the README on github. Can you provide a source?

Comment (Vaelatern) [inline srcpkgs/zerotier-one/template]: Source found.

**Pull request closed without merging**